addons/website_crm_partner_assign/views/res_partner_views.xml

```diff
@@ -23,7 +23,7 @@
             <field name="name">res.partner.activation.tree</field>
             <field name="model">res.partner.activation</field>
             <field name="arch" type="xml">
-                <tree string="Activation" editable="bottom">
+                <tree string="Activation" editable="bottom" default_order="sequence desc">
```

Comment:
Shouldn't we set that order directly at model level ? Is there something preventing it ?